.github/workflows/test.yaml

```diff
@@ -13,30 +13,27 @@ jobs:
   lint:
     name: "Lint"
     runs-on: ubuntu-latest
+    timeout-minutes: 5
+    permissions:
+      contents: read
     steps:
-      - uses: actions/checkout@v2
-      - name: golangci-lint
-        uses: golangci/golangci-lint-action@v2
```

Comment:
why remove the action?

Reply:
Still there in line [24](https://github.com/Azure/eraser/pull/74/files#diff-245392b692a50c38ecab4381b118862db514035c10983f3bd4f4b7f1f4be4692R24)

Reply:
let's keep the action (instead of make lint) since it has additional integration with github

Reply:
`make lint` is still good for local linting though

Reply:
Ok, I added it back but will keep the make lint for local use.